core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -800,9 +794,8 @@ import akka.stream.scaladsl.Source
           Extractors.optionalTaggedPersistentRepr(eventDeserializer, serialization))
         .mapAsync(1) { t =>
           t.tagged match {
-            case OptionVal.Some(tpr) =>
-              tr.sendMissingTagWrite(tp)(tpr)
             case OptionVal.None => FutureDone // no tags, skip
```

Comment:
IMO this was a bug (None is defined as `OptionalVal(null)`) and matching first on `Some(x)` would match `Some(null)` too.

Reply:
If it is a bug I would be a bit more comfortable changing this for 1.1.x rather than 1.1.0 (the premise being if it was critical bug it likely would have been solved by now).

@pjfanning @jrudolph @raboof wdyt?